docs/authentication.rst

```diff
@@ -3,18 +3,30 @@
 Authentication
 ==============
 
+This library offers several options for securing your server and authenticating
```

Comment:
Instead of saying 'server' (that makes me think of a hardware box), we should probably say 'service'.